README.md

```diff
@@ -47,7 +48,9 @@ As an example, lets say you want to edit the [Okta Angular Sign-in Widget](https
 To edit this page, you would navigate to `/packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md` to edit the content of this page.
 
 The directory structure from the `/packages/@okta/vuepress-site` folder relates directly to the URL you want to show. The `index.md` file that lives in the last folder in the structure will be rendered.
-If you name the file anything other than `index.md`, it will be required that you also include `.html` to the url when you go to view the page in the browser
+If you name the file anything other than `index.md`, it will be required that you also include `.html` to the url when you go to view the page in the browser.
+
+The [Guides](https://developer.okta.com/guides/) section has a more complex directory structure.  See the "Updating Guides" section below.
```

Comment:
Should we add a link to the "Updating Guides" section? I know the section is right below but it's still useful for lazy people like me that don't want to scroll :)